src/Athens-Examples/AthensTextRenderManualTests.class.st

```diff
@@ -113,7 +114,7 @@ AthensTextRenderManualTests class >> test4 [
 			canvas pathTransform translateBy: 0.1@10.
 		]
 	].
-	Display getCanvas translucentImage: surf asForm at: 100@100
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?